docs/ops/deployment/kubernetes.md

```diff
@@ -274,44 +341,121 @@ spec:
             path: flink-conf.yaml
           - key: log4j.properties
             path: log4j.properties
-
 {% endhighlight %}
 
-`jobmanager-service.yaml`
+### Job cluster resource definitions
+
+`jobmanager-job.yaml`
 {% highlight yaml %}
-apiVersion: v1
-kind: Service
+apiVersion: batch/v1
```

Comment:
Purely as a question for me; why are we using 3 different values for `apiVersion`?

Reply:
I am fully familiar with versioning of Kubernetes components but it matters. The versions correspond to the current examples on Kubernetes docs.